app/Repositories/ReportbackRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rogue\Repositories;
+
+use Rogue\Models\Reportback;
+use Rogue\Models\ReportbackLog;
+
+class ReportbackRepository
+{
+    /**
+     * Create a new reportback
+     *
+     * @param  array $data
+     * @return \Gladiator\Models\Message
+     */
+    public function create($data)
+    {
+        // Store reportback
+        $reportback = new Reportback;
```

Comment:
Totally not necessary, but you could combine these three lines into `$reportback = Reportback::create($data)`.